### ISSUE TITLE
'make test' fails in virtualenv on Mac OS X

### ISSUE THREAD

**Issue by ctb** [dib-lab/khmer#1263]
With latest master (94d679f2e2):
```
% make test
./setup.py develop
/Library/Frameworks/EPD64.framework/Versions/7.3/lib/python2.7/site-packages/setuptools-17.1.1-py2.7.egg/setuptools/dist.py:285: UserWarning: Normalizing '2.0-rc3+66.g94d679f' to '2.0rc3+66.g94d679f'
  normalized_version,
running develop
error: can't create or remove files in install directory

The following error occurred while trying to add or remove files in the
installation directory:

    [Errno 13] Permission denied: '/Library/Frameworks/EPD64.framework/Versions/7.3/lib/python2.7/site-packages/test-easy-install-9314.write-test'

The installation directory you specified (via --install-dir, --prefix, or
the distutils default setting) was:

    /Library/Frameworks/EPD64.framework/Versions/7.3/lib/python2.7/site-packages/

Perhaps your account does not have write access to this directory?  If the
installation directory is a system-owned directory, you may need to sign in
as the administrator or "root" account.  If you do not have administrative
access to this machine, you may wish to choose a different installation
directory, preferably one that is listed in your PYTHONPATH environment
variable.

For information on other options, you may wish to consult the
documentation at:

  https://pythonhosted.org/setuptools/easy_install.html

Please make the appropriate changes for your system and try again.

make: *** [test] Error 1
```

**Comment by ctb** [comment]
Note, 'make' works just fine, so it's specifically 'make test'.

I can also run 'make' and then run 'nosetests' ok.

**Comment by mr-c** [comment]
Everything works fine on our test machine. The first message shows that your virtualenv isn't activated.

**Comment by ctb** [comment]
The virtualenv is activated. This changed sometime in the last ~week.

**Comment by ctb** [comment]
039e5cc5f is broken; 28193b132a works (via git bisect).

**Comment by ctb** [comment]
It's caused by a difference in Makefile - see SHELL=bash addition to the Makefile, which has the effect of resetting the virtualenv. So I suggest removing it, yes? (ISTR we found this a long time ago, not sure how to find it in the source tree.)

**Comment by mr-c** [comment]
Wacky. I'm trying to reproduce this locally. Where are you experiencing this?

**Comment by ctb** [comment]
On Mon, Aug 17, 2015 at 06:12:08PM -0700, Michael R. Crusoe wrote:
> Wacky. I'm trying to reproduce this locally. Where are you experiencing this?

On my laptop.

**Comment by mr-c** [comment]
Which is running Mavericks, okay. I'll see about de-BASHing the Makefile & I'll add a note.

**Comment by mr-c** [comment]
Fixed in #1266
